components/brave_wallet_ui/page/screens/dev-bitcoin/dev-bitcoin.tsx

```diff
@@ -0,0 +1,251 @@
+import { useSelector, useDispatch } from 'react-redux'
+import { WalletActions } from '../../../common/actions'
+
+import styled from 'styled-components'
+import { useState } from 'react'
+import { BraveWallet, WalletState } from '../../../constants/types'
+import { LoadingSkeleton } from '../../../components/shared'
+
+const StyledWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: flex-start;
+  width: 100%;
+  padding-top: 32px;
+`
+
+const AccountInfoSection = styled.div``
+
+const AddressesHeader = styled.h2`
+  margin-bottom: 0;
+`
+
+const AddressLine = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: start;
+`
+
+const Address = styled.div`
+  font-family: monospace;
+  margin-right: 30px;
+`
+
+const Balance = styled.div`
+  font-family: monospace;
+`
+
+const SendToSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: start;
+`
+
+const SendToLine = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+`
+
+const defaultNetworkId = BraveWallet.BITCOIN_TESTNET
+const defaultKeyringId = BraveWallet.BITCOIN_KEYRING84_TEST_ID
+const accountIndex = 0
+
+const CreateAccountSection = () => {
+  const dispatch = useDispatch()
+
+  const createBtcAccount = async () => {
+    dispatch(
+      WalletActions.addBitcoinAccount({
+        accountName: 'BTC Account',
+        networkId: defaultNetworkId,
+        keyringId: defaultKeyringId
+      })
+    )
+  }
+
+  return (
+    <StyledWrapper>
+      <button onClick={createBtcAccount}>Create Account</button>
+    </StyledWrapper>
+  )
+}
+
+const AccountSection = () => {
+  const [loading, setLoading] = useState<boolean>(true)
+
+  const [accountInfo, setAccountInfo] =
+    useState<BraveWallet.BitcoinAccountInfo | null>(null)
+
+  const [sendToAddress, setSendToAddress] = useState<string>('')
+
+  const [sendToAmount, setSendToAmount] = useState<number>(1000)
+  const [sendToFee, setSendToFee] = useState<number>(300)
+  const [sending, setSending] = useState<boolean>(false)
+  const [sendResult, setSendResult] = useState<{
+    txid: string
+    error: string
+  } | null>(null)
+
+  const handlesendToAddressInputChanged = ({
+    target: { value }
+  }: React.ChangeEvent<HTMLInputElement>) => {
+    setSendToAddress(value)
+  }
+
+  const handlesendToAmountInputChanged = ({
+    target: { value }
+  }: React.ChangeEvent<HTMLInputElement>) => {
+    setSendToAmount(Number(value))
+  }
+
+  const handlesendToFeeInputChanged = ({
+    target: { value }
+  }: React.ChangeEvent<HTMLInputElement>) => {
+    setSendToFee(Number(value))
+  }
+
+  const fetchAccountInfo = async () => {
+    const accountInfo =
+      await getAPIProxy().bitcoinWalletService.getBitcoinAccountInfo(
+        defaultNetworkId,
+        defaultKeyringId,
+        accountIndex
+      )
```

Comment:
a query should be defined with `api.slice.ts` to simplify fetching & caching of this data